### PR TITLE
Symbol table fixes

### PR DESCRIPTION
**Pull request by Therzok**

### CODE REVIEW

File: src/Gir/Marshal/SymbolTable.Builtin.cs [hunk 1]
```diff
@@ -41,7 +40,42 @@ void RegisterPrimitives ()
...
+			//AddType(new MarshalGen("time_t", "System.DateTime", "IntPtr", "GLib.Marshaller.DateTimeTotime_t ({0})", "GLib.Marshaller.time_tToDateTime ({0})"));
+
+			//AddType(new ConstStringGen("const-gchar"));
+			//AddType(new ConstStringGen("const-xmlChar"));
+			//AddType(new ConstStringGen("const-char"));
+			//AddType(new ConstFilenameGen("const-gfilename"));
+			//AddType(new StringMarshalGen("gfilename", "string", "IntPtr", "GLib.Marshaller.StringToFilenamePtr({0})", "GLib.Marshaller.FilenamePtrToStringGFree({0})"));
+			//AddType(new StringMarshalGen("gchar", "string", "IntPtr", "GLib.Marshaller.StringToPtrGStrdup({0})", "GLib.Marshaller.PtrToStringGFree({0})"));
+			//AddType(new StringMarshalGen("char", "string", "IntPtr", "GLib.Marshaller.StringToPtrGStrdup({0})", "GLib.Marshaller.PtrToStringGFree({0})"));
+			//AddType(new SimpleGen("GStrv", "string[]", "null"));
```
Comment (stsundermann) [inline src/Gir/Marshal/SymbolTable.Builtin.cs]: This one is aliased

File: src/Gir/Marshal/SymbolTable.Builtin.cs [hunk 2]
```diff
@@ -41,7 +40,42 @@ void RegisterPrimitives ()
...
+
+			AddType(new Primitive("ssize_t", "IntPtr", "IntPtr.Zero"));
+			AddType(new Primitive("gssize", "IntPtr", "IntPtr.Zero"));
+			AddType(new Primitive("size_t", "UIntPtr", "UIntPtr.Zero"));
+			AddType(new Primitive("gsize", "UIntPtr", "UIntPtr.Zero"));
+
+			// FIXME: Add these eventually.
+			//AddType(new MarshalGen("time_t", "System.DateTime", "IntPtr", "GLib.Marshaller.DateTimeTotime_t ({0})", "GLib.Marshaller.time_tToDateTime ({0})"));
+
+			//AddType(new ConstStringGen("const-gchar"));
```
Comment (stsundermann) [inline src/Gir/Marshal/SymbolTable.Builtin.cs]: Not sure about "const-" prefix here

Comment (Therzok) [reply src/Gir/Marshal/SymbolTable.Builtin.cs]: It's commented out. Needs to be fixed when imported with proper marshalling.

Comment (stsundermann) [reply src/Gir/Marshal/SymbolTable.Builtin.cs]: Yeah, I was just making sure we do not forget about that ;)

**Pull request merged**